src/components/App.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import ButtonPanel from './ButtonPanel';
+import Display from './Display';
+import '../App.css';
+
+const App = () => (
+  <>
+    <Display result="0" />
```

Comment:
- [ ] We do not need to pass the result props when it's 0 after setting it a default value for it.